pyanaconda/ui/gui/spokes/custom.py

```diff
@@ -2628,6 +2628,10 @@ def on_info_bar_clicked(self, *args):
     @timed_action(delay=50, threshold=100)
     def on_update_settings_clicked(self, button):
         """ call _save_right_side, then, perhaps, populate_right_side. """
+        if button:
```

Comment:
In what case the button won't be present?

Reply:
Sometimes we call the method directly with `None` and it is difficult to test if it is ok to clear the errors at that points. This code is enough to fix the bug, but it shouldn't change anything else.

Reply:
When this is targeted to RHEL, then it should be fine with this patch. However, backport of this patch should be reworked to remove calls without button to this method.